prototype/Character/Player/Player.gd

```diff
@@ -159,6 +159,9 @@ func _integrate_forces(state):
 	if Input.is_action_just_pressed('toggle_jetpack') and allow_jetpack:
```

Comment:
Is `toggle_jetpack` still a thing with this change? If not, why not remove related code?

Reply:
It's still a thing. Maybe I'm just used to the old controls, but you may want to de-toggle it manually.